hw3/01-characters.js

```diff
@@ -1,4 +1,24 @@
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
+const section = document.querySelector('section');
 
-fetch(url);
+fetch(url)
+  .then((response) => response.json())
+  .then((data) => {
+    for (const character of data) {
```

Comment:
I wasn't familiar with looping like this and not sure if it was standard so I checked Airbnb guidelines and it looks like they recommend using array methods like `forEach`

https://github.com/airbnb/javascript#iterators--nope